pkgs/by-name/sl/sleek-grub-theme/package.nix

```
@@ -0,0 +1,39 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, withBanner ? "Grub Bootloader" # use override to specify your own banner text
+, style ? "white" # use override to specify one of "dark" / "orange" / "bigSur"
```

```suggestion
, withStyle ? "white" # use override to specify one of "dark" / "orange" / "bigSur"
```

Just to be consistent here.

---

I think `banner` and `style` make more sense.

---

I guess that didn't work because of [the `banner` package](https://github.com/NixOS/nixpkgs/blob/master/pkgs/games/banner/default.nix).